src/Altinn.App.Core/Features/Correspondence/Models/SendCorrespondenceResponse.cs

```diff
@@ -0,0 +1,21 @@
+using System.Text.Json.Serialization;
+
+namespace Altinn.App.Core.Features.Correspondence.Models;
+
+/// <summary>
+/// Response after a successful <see cref="CorrespondenceClient.Send"/> request
+/// </summary>
+public sealed record SendCorrespondenceResponse
+{
+    /// <summary>
+    /// The correspondences that were processed
+    /// </summary>
+    [JsonPropertyName("correspondences")]
+    public required List<CorrespondenceDetailsResponse> Correspondences { get; init; }
```

Comment:
For responses, which generally are immutable, maybe we should use `IReadOnlyList` or something instead?